go.mod

```diff
@@ -1,19 +1,20 @@
 module fuse-device-plugin
 
-go 1.24
+go 1.24.0
+
+toolchain go1.24.1
```

Comment:
So we want 1.24.0 as minimum but want to force building with 1.24.1 ?

Reply:
I think I'd prefer to directly set `go 1.24.1` because this is not a library. What is the advantage of using `toolchain`?